compose.yaml

```diff
@@ -5,15 +5,20 @@ services:
     init: true
     build:
       context: .
+      target: development
+    env_file: .env.docker
     environment:
-      API_URI: ${API_URI}
-      DATABASE_URL: postgres://${PG_USER}:${PG_PASSWORD}@database:5432/${PG_DATABASE}
+      API_URI: ${API_URI-http://localhost:2300}
+      DATABASE_URL: postgres://${PG_USER:-postgres}:${PG_PASSWORD:-password}@database:5432/${PG_DATABASE:-terminus_development}
     ports:
       - "2300:2300"
     restart: unless-stopped
+    volumes:
+      - .:/app
```

Comment:
:axe: I'm afraid this'll cause issues in a production environment with the app folder being cached like this. Plus, the `Dockerfile` already copies over the relevant application files.